config/services/build/generate.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Files generate.xml and transfer.xml just examples of how ece-tools commands can be configured by Symfony DI.
But at the moment this has no sense as they are configured in scneario*.xml files.